test-interface/src/main/scala/sbt/testing/Event.scala

```diff
@@ -1,41 +1,40 @@
 package sbt.testing
 
-/**
- * An event fired by the test framework during a run.
- */
+/** An event fired by the test framework during a run. */
 trait Event {
 
-  /**
-   * The fully qualified name of a class that can rerun the suite or test
-   * about which an event was fired.
+  /** The fully qualified name of a class that can rerun the suite or test
+   *  about which an event was fired.
```

Comment:
Does scalacheck enforce this? Otherwise we might want to move to ScalaDoc comments:

```scala
/** Foo
  * Bar
  */
```